web/components/answers/answers-panel.tsx

```diff
@@ -383,6 +394,20 @@ function Answer(props: {
   const hasBets = userBets && !floatingEqual(sharesSum, 0)
   const isMobile = useIsMobile()
 
+  const limitOrderVolume = useMemo(
+    () =>
+      unfilledBets?.reduce((acc, bet) => {
+        acc += bet.orderAmount
+
+        bet.fills.map((fill) => {
+          acc -= fill.amount
+        })
```

Comment:
I think the sum of the fills is always `bet.amount`. So you could just subtract that.

Reply:
I was trying to account for partial fills but maybe thats not necessary? I was just trying to get the amount of mana left tied up in limit orders but maybe I am calculating it wrong.

Reply:
Yeah, this is right. It's just `bet.orderAmount - bet.amount`

Reply:
Right on! Just pushed a fix.